script.js

```diff
@@ -0,0 +1,157 @@
+
+const userEntry = document.querySelector('#display');
+const btnContainer = document.querySelector('#button-container');
+const operBtns = document.querySelectorAll('.oper');
+const backspace = document.querySelector('#backspace');
+
+//handles keyboard entries
+window.addEventListener('keydown', event => {
+    if (!isNaN(event.key)) userEntry.textContent += event.key;
+
+    if (oper.hasOwnProperty(event.key)) {
+        userEntry.textContent += event.key;
+    };
+
+    if (event.key === '/') userEntry.textContent += '÷';
```

Comment:
reduce complexity, why convert / to divide? dont even have the key on my keyboard, if someone were refactoring and/or extending this it would make the process unnecessarily frustrating